src/main/resources/explanations/challenge34_hint.adoc

```diff
@@ -0,0 +1,6 @@
+Like other Git challenges this can be solved by manually combing the Git metadata. 
+There are also a plethora of tools that will automatically search for secrets leaked in Git repos.
+
+1. Search manually using `git log`. You can then use a tool such as grep to try and find leaked secrets with regex searches.
```

Comment:
Would it be possible to give the steps here like we do on earlier assignments?
- clone the repo
- ...

Reply:
Sure, i can make them more verbose :) Do you have an example of a good hint doc?

Reply:
https://github.com/OWASP/wrongsecrets/blob/3a479889de50cc23551c2a2a98c565b7560b2253/src/main/resources/explanations/challenge1_hint.adoc#L4